plugins/diff/erofs/differ.go

```diff
@@ -100,11 +98,7 @@ func NewErofsDiffer(store content.Store, opts ...DifferOpt) differ {
 // Since `images.DiffCompression` doesn't support arbitrary media types,
 // disallow non-empty suffixes for now.
 func isErofsMediaType(mt string) bool {
-	mediaType, _, hasExt := strings.Cut(mt, "+")
-	if hasExt {
```

Comment:
maybe leave `hasExt` logic as-is since the erofs differ doesn't implement  non-empty suffixes for now and  https://github.com/containerd/containerd/pull/12506 addresses it.